dj_db_conn_pool/backends/mysql/base.py

```diff
@@ -1,7 +1,7 @@
 import logging
 
 from django.db.backends.mysql import base
-from sqlalchemy.dialects.mysql.base import MySQLDialect
+from sqlalchemy.dialects.mysql.mysqldb import MySQLDialect_mysqldb as MySQLDialect
```

Comment:
Django supports mysqlclient and pymysql as a MySQL driver officially, though, this library supports mysqlcilent only (currently at least).
So, I decide to use `MySQLDialect_mysqldb` directly instead of using both `MySQLDialect_mysqldb` and `MySQLDialect_pymysql` selectively following the conditions.